.gitignore

```diff
@@ -0,0 +1,32 @@
+# Created by .ignore support plugin (hsz.mobi)
```

Comment:
If you've made general improvements to this file that would be useful to `prod-module-boilerplate`, could you make sure they're reflected there, too? I haven't looked myself...